tasks/pre_install_checks.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Gather facts on installed packages
+  package_facts:
+    manager: rpm
+  no_log: true
+
+- name: Fail when firewall manager is not installed
+  fail:
+    msg: '{{ ovirt_engine_setup_firewall_manager }} was chosen as a firewall manager but is not installed'
+  when:
+    - ovirt_engine_setup_firewall_manager not in ansible_facts.packages
+    - ovirt_engine_setup_firewall_manager|bool
```

Comment:
remove the `|bool` (it converts `"True"` to `True`)

Reply:
Removing `bool` triggers the following warning when `ovirt_engine_setup_firewall_manager` is `null`:
```
[DEPRECATION WARNING]: evaluating None as a bare variable, this behaviour will go away and you might need to add |bool to the expression in the future. Also see CONDITIONAL_BARE_VARS 
configuration toggle.. This feature will be removed in version 2.12. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```
Are wa okay with that?

Reply:
Oh did not know about this use case with null... better to keep the bool

Reply:
done some testing and think it would break the logic because if there is some other value then "True" it's converted to False, so when you would put for example firewalld it would not pass

try to use `is not none`

Reply:
The check for `is not none` seems to be working nice. I ran a series of tests, see here: https://pastebin.com/UmmEYG2x

During the tests, I disabled most of the role's execution and kept only tasks/pre_install_checks.yml. All of the different cases now behave as we expect. The only one that's a little quirky is the last one - empty string. It fails because "" of course is not among installed packages. However, I think that's fine because we instruct users to provide empty value if they want to skip the firewall configuration.